creator-node/src/components/healthCheck/healthCheckComponentService.js

```diff
@@ -100,6 +101,12 @@ const healthCheck = async (
 
   const asyncProcessingQueueJobs = await getAsyncProcessingQueueJobs()
 
+  const isAvailable = await transcodingQueueIsAvailable()
+  const shouldHandleTranscode = utils.currentNodeShouldHandleTranscode({
+    transcodingQueueCanAcceptMoreJobs: isAvailable,
+    spID: config.get('spID')
```

Comment:
I think this all looks good! nothing to do here, but just a callout that I've noticed our usage of `spID` vs `spId` is inconsistent throughout the codebase -- maybe we should have a task to standardize this?

Reply:
yes good call out! one of the many things we need to standardize 😅